includes/services/extended/0-google-base.php

```diff
@@ -29,9 +29,11 @@ function __construct() {
 		$this->set_endpoint( 'userinfo', 'https://www.googleapis.com/oauth2/v3/userinfo', 'GET' );
 
 		$creds              = $this->get_credentials();
-		$this->redirect_uri = $creds['redirect_uri'];
-		$this->key          = $creds['key'];
-		$this->secret       = $creds['secret'];
+		if ( ! empty( $creds ) ) {
+			$this->redirect_uri = $creds['redirect_uri'];
```

Comment:
If we're checking each of the properties in `init_credentials` should we add `redirect_uri` to it so we can call that function here?

Reply:
Sure, I wasn't sure about trying to deal with the different requirements for each one but that one is pretty close.  Changed in 06eb8b5dca2efe3ea8ffb4941f2b05aa472787c0